src/pages/components/progress-bar/usage.mdx

```diff
@@ -0,0 +1,504 @@
+developers will have to calculate the percentage on their side with the value:
+`${(value/max*100).toFixed(2)}% (${value} of ${max} items)`
+
+| Variant                        | Purpose                                                                                                                                                                                                                                                    |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
+| [Determinate](Determinate)     | Use determinate progress bars for a specified amount of time or clear information about the progression — they convey exact and qualitative information about the progression.                                                                             |     |
+| [Indeterminate](Indeterminate) | Use indeterminate progress bars for an unspecified completion of time or unclear information about the progression — they convey that users’ request, action, or data is being processed at the moment without indicating how long the activity will take. |
+
+<Row>
+<Column colLg={4}>
+
+![Determinate progress bars fill from 0 to 100%.](images/progress-bar-usage-2.gif)
+
+<Caption>Determinate progress bars fill from 0 to 100%.</Caption>
+
+</Column>
+
+<Column colLg={4}>
+
+![Indeterminate progress bars move along a fixed track continually until the process is complete.](images/progress-bar-usage-3.gif)
+
+<Caption>
+  Indeterminate progress bars move along a fixed track continually until the
+  process is complete.
+</Caption>
+
+</Column>
+</Row>
+
+## Determinate progress bar
+
+Determinate progress bars are used when the loading progress can be tracked and
+translated into a percentage indicator which fills from 0% to 100% and never
+decrease in value. The entire animation moves from the left to the right – with
+no back and forth motion.
+
+Use a determinate progress bar when progress can be calculated against a
+specific goal. For example, when downloading a file of a known size.
+
+<Row>
+<Column colLg={8}>
+
+![Progress bar uploading](images/progress-bar-usage-4.gif)
+
+</Column>
+</Row>
+
+## Indeterminate progress bar
+
+Once triggered, the loading bar will begin by moving along in constant speed,
+repeating over time while data is being gathered during the first stages of the
+load. This bar then changes to a determinate indicator (finishing from left to
+right) when enough information is gathered to end the load animation. The final
+determinate stage of the load is used to visually indicate the completion of the
+task to the user. In this scenario, a percentage indicator would not be present.
+
+<Row>
+<Column colLg={8}>
+
+![Progress bar uploading](images/progress-bar-usage-5.gif)
```

Comment:
This example isn't changing to a determinate indicator in second stage like the paragraph says.